### PR TITLE
[Sanity] add caps/MTL for paltform MTL-P

### PR DESCRIPTION
**Pull request by Wanghangjie8**
Signed-off-by: Wang Hangjie <hangjiex.wang@intel.com>

### CODE REVIEW

**Comment by Wanghangjie8** [comment]
I added it based on this information.
https://github.com/intel/media-driver#:~:text=Decoding/Encoding%20Features

**Comment by Wanghangjie8** [comment]
@FocusLuo @uartie can you review this for me?

File: lib/caps/MTL/iHD [hunk 1]
```diff
@@ -0,0 +1,98 @@
+###
+### Copyright (C) 2021 Intel Corporation
```
Comment (uartie) [inline lib/caps/MTL/iHD:2]: fix copyright year

File: lib/caps/MTL/info [hunk 2]
```diff
@@ -0,0 +1,13 @@
+###
+### Copyright (C) 2021 Intel Corporation
```
Comment (uartie) [inline lib/caps/MTL/info:2]: fix copyright year

File: lib/caps/MTL/info [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+###
+### kate: syntax python;
+###
+
+info = dict(
+  gpu = dict(gen = 12.2),
```
Comment (uartie) [inline lib/caps/MTL/info:12]: Is this the correct gpu gen?  @FocusLuo can you confirm?

**Review by uartie — CHANGES_REQUESTED** [review]

File: lib/caps/MTL/info (hunk 3 above)
Comment (Wanghangjie8) [reply lib/caps/MTL/info]: updated

**Pull request merged**